### ISSUE TITLE
Fork that works on NextCloud 27

### ISSUE THREAD

**Issue by devnoname120** [e-alfred/epubreader#51]
**See**: https://github.com/devnoname120/epubviewer
Submission to the NextCloud App Store is in progress.

cc @sysadminstory @fcki1984 @NiklasSchmitt @shyzus @fossil-free @Emiliaaah @markuman @DerBoki

**Comment by jdoss2020** [comment]
This new plugin having problems on Nextcloud 27.0.2.  When I install and enable the epub 1.5 app, I cannot see my files under the files tab. If I bounce back and forth between the left hand menu options, I can eventually see the directories under Files, but clicking on the directories does nothing. When I uninstall and remove the epub 1.5 app and bounce the docker container, the behavior under files returns to normal. I would really love to have a working epub reader under nextcloud so I hope you can resolve this issue.

**Comment by devnoname120** [comment]
@jdoss2020 Can you paste the error and the log?

**Comment by jdoss2020** [comment]
The details should be on this thread.......

https://github.com/devnoname120/epubviewer/issues/3

**Comment by jdoss2020** [comment]
Any progress on this issue?

**Comment by devnoname120** [comment]
@jdoss2020 Yes, it's fixed in version 1.5.2. It's published in the app store too: https://apps.nextcloud.com/apps/epubviewer

**Comment by thenamelessthing** [comment]
@devnoname120 can you help me on this?

https://github.com/e-alfred/epubreader/issues/46

**Comment by devnoname120** [comment]
@thenamelessthing Do a PR

**Comment by thenamelessthing** [comment]
> @thenamelessthing Do a PR

I tried to make a pull request but it doesn't work. So I continue to document my problem here. I think I've figured out the problem pretty well.

See https://github.com/e-alfred/epubreader/issues/46
